be/src/exprs/jit/ir_helper.h

```diff
@@ -102,6 +107,8 @@ class IRHelper {
     static llvm::Value* build_if_else(llvm::Value* condition, llvm::Type* return_type,
                                       const std::function<llvm::Value*()>& then_func,
                                       const std::function<llvm::Value*()>& else_func, llvm::IRBuilder<>* builder);
+
+    static constexpr double jit_score_ratio = 0.88; // whether the expr can be jit
```

Comment:
should we make this variable configurable?

Reply:
prefer not, as there isn't a ideal value for all cases. If the adaptive jit not work, the user can force enable jit.